doc/installation-and-configuration.md

```diff
@@ -91,18 +91,10 @@ While preparing a Virtual Machine that will host SonarQube database, portal and/
  
 		![](_img/SonarQube-Port-9090.png)
 
-		**>>NOTE >>** Before proceeding with the below configuration steps make sure you have configured SonarQube to use SQL Server database instead of embedded database. 
-	
-	- Search for and locate the entry for **sonar.jdbc.username**.
-
-		![](_img/Sonar.jdbc.username-entry.png)
-	- Uncomment (i.e. delete the leading ‘\#’) the two **sonar.jdbc** settings circled in the screenshot above and replace **sonar** in each setting with the database login name and password, respectively.
-
-		![](_img/sonar.jdbc-delete-leading-hash.png)
 	- Search for and locate the entry for sonar.jdbc.url. There are several copies of this setting based on database type. Make sure you select the entry for Microsoft SQL Server.
 
-		![](_img/sonar.jdbc-sqlserver.png)
-	- Uncomment (i.e. delete the leading ‘\#’) the sonar.jdbc.url setting circled in the screenshot above and replace the connection string to match the server\\instance and database name for your machine. Example: **sqlserver://.\\SQLExpress/Sonar;SelectMethod=Cursor**
+		![](_img/sonar.ms-jdbc-sqlserver.png)
+	- Uncomment (i.e. delete the leading ‘\#’) the sonar.jdbc.url setting circled in the screenshot above and replace the connection string to match the server and database name for your machine. Example: **sqlserver://localhost;databaseName=sonar;
 

```

Comment:
Is there a minimum version of the SQ server that supports integrated security? Should we inform the user about this?